third_party/blink/renderer/core/dom/weak_identifier_map.h

```diff
@@ -43,6 +43,10 @@ class WeakIdentifierMap final
     return result;
   }
 
+  static IdentifierType HasIdentifier(T* object) {
```

Comment:
This signature looks like it should be `static bool HasIdentifier(T* object)` instead.